.travis.yml

```
@@ -53,11 +53,20 @@ before_install:
     fi
 install: true
 script: |
-        if [ -z "${UNENCRYPTED_PACKAGE:-}" ]; then
-        citus_package -p ${TARGET_PLATFORM} 'local' 'release'
+        if [ -z "${UNENCRYPTED_PACKAGE:-}" ] && [ ${TRAVIS_EVENT_TYPE} != 'cron' ] ; then
+          citus_package -p ${TARGET_PLATFORM} 'local' 'release'
+        elif [ -z "${UNENCRYPTED_PACKAGE:-}" ] && [ ${TRAVIS_EVENT_TYPE} == 'cron' ] ; then
+          citus_package -p ${TARGET_PLATFORM} 'local' 'nightly'
         else
-        build_new_release && build_new_nightly
+          if [ ${TRAVIS_EVENT_TYPE} == 'push' ] || [ ${TRAVIS_EVENT_TYPE} == 'pull_request' ] || [ ${TRAVIS_EVENT_TYPE} == 'api' ]; then
+            echo "Travis Event Type is ${TRAVIS_EVENT_TYPE}"
+            build_new_release
+          elif [ ${TRAVIS_EVENT_TYPE} == 'cron' ]; then
+            echo "Travis Event Type is ${TRAVIS_EVENT_TYPE}"
+            build_new_nightly
+          fi
```

minor: could have only one `echo`  line:

```suggestion
          echo "Travis Event Type is ${TRAVIS_EVENT_TYPE}"
          if [ ${TRAVIS_EVENT_TYPE} == 'push' ] || [ ${TRAVIS_EVENT_TYPE} == 'pull_request' ] || [ ${TRAVIS_EVENT_TYPE} == 'api' ]; then
            build_new_release
          elif [ ${TRAVIS_EVENT_TYPE} == 'cron' ]; then
            build_new_nightly
          fi
```